core/server/common/src/main/java/alluxio/master/audit/AsyncUserAccessAuditLogWriter.java

```diff
@@ -103,6 +103,14 @@ public boolean append(AuditContext context) {
     return true;
   }
 
+  /**
+   * Gets the audit log entries.
+   * @return the blocking queue of audit log
+   */
+  public LinkedBlockingQueue<AuditContext> getAuditLogEntries() {
```

Comment:
Create a following method instead to avoid expose the mAuditLogEntries out of this class.

```java
public long getAuditLoggingQueueSize() {
  return mAuditLogEntries.size();
}
```